### PR TITLE
Remove configuration examples

### PR DESCRIPTION
**Pull request by acsfer**
Values are very specific for each system.
Fix https://github.com/nextcloud/documentation/issues/6046

### CODE REVIEW

**Review by kesselb — APPROVED** [review]
Thanks :+1:

**Review by MorrisJobke — APPROVED** [review]

File: admin_manual/installation/server_tuning.rst [hunk 1]
```diff
@@ -95,35 +83,21 @@ AES-NI extension:
 * If your environment runs virtualized, check the virtualization vendor for
   support.
 
-Enable HTTP2 for faster loading
+Enable HTTP/2 for faster loading
 -------------------------------
```
Comment (nickvergessen) [inline admin_manual/installation/server_tuning.rst:87]:
```
--------------------------------
```

Comment (MorrisJobke) [reply admin_manual/installation/server_tuning.rst:87]: 419a126f339be7c37d8b03e9fd2a79fb24042f0f

Comment (acsfer) [reply admin_manual/installation/server_tuning.rst:87]: 🐘

Comment (MorrisJobke) [reply admin_manual/installation/server_tuning.rst:87]: 🦆

**Pull request merged**